components/automate-ui/src/app/entities/node-credentials/node-credential.model.ts

```diff
@@ -37,6 +38,7 @@ export interface NodeObject {
   sudo: {password: string, options: string};
 }
 
+@Injectable()
```

Comment:
In Angular 10, classes that use Angular features and do not have an Angular decorator are no longer supported.  `ng update` migrates this automatically.